conda/resolve.py

```diff
@@ -671,15 +671,13 @@ def generate_feature_count(self, C, trackers):
 
     def generate_feature_metric(self, C, groups, specs):
         eq = {}
+        total = 0
         for name, group in iteritems(groups):
             nf = [len(self.features(fn)) for fn in group]
             maxf = max(nf)
```

Comment:
Oh I'm not worried about the performance of the function itself, rather the performance of the SAT solver that uses its output.